network_analysis/find-route/README.md

```diff
@@ -1,36 +1,35 @@
-# Find Route
+# Find route
 
-Get a route between two stops with directions.
+Display directions for a route between two points.
 
-![](FindRoute.png)
+![Image of find route](FindRoute.png)
+
+## Use case
+
+Find routes with driving directions between any number of locations. You might use the ArcGIS platform to create a custom network for routing on a private roads.
 
 ## How to use the sample
 
-For simplicity, the sample comes loaded with a start and end stop. You can click on the Find route button to get a route between these stops. Once the route is generated, the DirectionMessages show step by step directions in the directions list.
+For simplicity, the sample comes loaded with a start and end stop. You can tap on the 'Find Route' button to display a route between these stops. Once the route is generated, turn-by-turn directions are shown in a list. You can use the 'Reset' button to reset the sample.
```

Comment:
we've been using double quotes elsewhere for buttons so might as well continue that!
```suggestion
For simplicity, the sample comes loaded with a start and end stop. You can tap on the "Find Route" button to display a route between these stops. Once the route is generated, turn-by-turn directions are shown in a list. You can use the "Reset" button to reset the sample.
```